src/schemas/fairdata.bundle.js

```diff
@@ -55,7 +55,7 @@ export default {
 	ida: {
 		schema: FairdataIdaSchema,
 		ui: FairdataIdaUi,
-		name: "I want to select IDA files", // "Fairdata (IDA)",
+		name: "Select IDA files", // "Fairdata (IDA)",
```

Comment:
this is out of scope of this issue.

Reply:
I'd say that having an unnecessarily long text in a field affects usability when space is limited, which is why I did this here, but it's not a problem to move these changes to a new ticket. I'll do that.

Now that I think about it, the reason the schemas have a "name" (shown before schema is selected) and "shortName" (shown after schema is selected) is that the previous toolbar layout often didn't show what schema you had selected because there was only enough space to see the first few characters. Now that the toolbars show the selected dropdown option better, maybe shortNames should be removed so the options would have consistent names that don't change. I'll add that to the ticket.

Reply:
My concern is that this changes the end user experience and the user manuals/guides as well as tests. So it makes sense to track this kind of change in a separate issue and a pull request.

Reply:
That's a good point for changes that affect in the UI. I'll ping Erja in the ticket to make sure the change is ok.

Reply:
Removed the out-of-scope parts, will make a new PR for them.